src/main/kotlin/no/nav/personbruker/dittnav/api/brukernotifikasjon/brukernotifikasjonApi.kt

```diff
@@ -0,0 +1,46 @@
+package no.nav.personbruker.dittnav.api.brukernotifikasjon
+
+import io.ktor.application.call
+import io.ktor.http.HttpStatusCode
+import io.ktor.response.respond
+import io.ktor.routing.Route
+import io.ktor.routing.get
+import no.nav.personbruker.dittnav.api.common.respondWithError
+import no.nav.personbruker.dittnav.api.config.innloggetBruker
+import org.slf4j.LoggerFactory
+
+fun Route.brukernotifikasjoner(service: BrukernotifikasjonService) {
+
+    val log = LoggerFactory.getLogger(BrukernotifikasjonService::class.java)
+
+    get("/brukernotifikasjon/count") {
```

Comment:
Er det noen grunn til at i handler er endepunktet count/brukernotifikasjoner, mens her brukernotifikasjon/count? D.v.s. annen rekkefølge.

Reply:
Jeg gjorde det slik for at det skulle bli likt mønsteret som var brukt i her app. Men det bør vi ta en diskusjon om, slik at vi har en enhetlig måte å gjøre dette på 😃